docs/en/agent/support-plugins.md

```diff
@@ -8,9 +8,10 @@ metrics based on the tracing data.
   * `go-restfulv3`: [Go-Restful](https://github.com/emicklei/go-restful) tested v3.7.1 to 3.10.2.
   * `mux`: [Mux](https://github.com/gorilla/mux) tested v1.7.0 to v1.8.0.
   * `iris`: [Iris](https://github.com/kataras/iris) tested v12.1.0 to 12.2.5.
+  * `fasthttp`: [FastHttp](https://github.com/valyala/fasthttp) tested every ten versions from v1.0.0 to v1.50.0.
```

Comment:
What do you mean every ten version? Do you mean 1.0, 1.10, 1.20?

Reply:
Yes, should i change the instructions here

Reply:
You could simply say 1.0.0 to 1.50.

Reply:
Got it.
it has been changed.